cuda_core/docs/source/release/0.1.1-notes.md

```diff
@@ -1,25 +1,28 @@
-# `cuda.core` Release notes
+# `cuda.core` v0.1.1 Release notes
 
-Released on Dec XX, 2024
+Released on Dec 20, 2024
 
 ## Hightlights
 
 - Add `StridedMemoryView` and `@args_viewable_as_strided_memory` that provide a concrete
   implementation of DLPack & CUDA Array Interface supports.
 - Add `Linker` that can link one or multiple `ObjectCode` instances generated by `Program`s. Under
-  the hood, it uses either the nvJitLink or cuLink APIs depending on the CUDA version detected
```

Comment:
I think the we can leave 'Program' singular, but I don't have a strong preference

Reply:
done in commit [3641e27](https://github.com/NVIDIA/cuda-python/pull/301/commits/3641e2730efcb0fc63c57e1a406c57e207fde102)